internal/tiltfile/k8s_custom_deploy.go

```diff
@@ -149,3 +151,24 @@ func (s *tiltfileState) k8sCustomDeploy(thread *starlark.Thread, fn *starlark.Bu
 
 	return starlark.None, nil
 }
+
+func filterDepsForCustomDeploy(deps []string, spec v1alpha1.LiveUpdateSpec) (deployPaths []string, luPaths []string) {
+	luMatcher := pathMatcherFromLiveUpdateSpec(spec)
+	for _, dep := range deps {
+		if match, _ := luMatcher.Matches(dep); match {
```

Comment:
i don't think this is quite right, since you could have:

k8s_custom_deploy(deps=['.'], live_update=[sync('./src', ...)])

i think you probably need to add the live update paths as an ignore path?

Reply:
Ahhh great point. Changed things around in 98cf10ea9, integration test now uses `deps=['.']` and passes 👍